.ebextensions/99datadog.config

```diff
@@ -21,6 +21,10 @@ files:
       apm_config:
         enabled: "true"
         apm_non_local_traffic: "true"
+      # Enable process_collection to see the cpu and memory usage of individual processes
+      process_config:
+        process_collection:
+          enabled: "false"
```

Comment:
I'm leaving this in turned off because it might be useful in future, but it costs an extra $9 per instance per month